### PR TITLE
fix ci tests

### PR DESCRIPTION
**Pull request by mrkishi**
### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [ ] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

File: packages/kit/src/core/preview/index.js [hunk 1]
```diff
@@ -161,7 +161,7 @@ export async function preview({ port, host, config, https: use_https = false })
 	});
 
 	return new Promise((fulfil) => {
-		http_server.listen(port, host || '0.0.0.0', () => {
+		http_server.listen(port, host, () => {
```
Comment (Conduitry) [inline packages/kit/src/core/preview/index.js:164]: Just to confirm: When the user passes `--host` with no argument (which indicates they want to bind on all interfaces), `host` will be an empty string, and passing `''` to `.listen` will have that effect?

Comment (mrkishi) [reply packages/kit/src/core/preview/index.js:164]: Passing `''` would be equivalent to `undefined` and results in `0.0.0.0` or `::`, depending on the network support (and I just tested to confirm it). The difference in this case is that we will bind all ipv6 and ipv4 interfaces, whereas before we only did ipv4.

Comment (Conduitry) [reply packages/kit/src/core/preview/index.js:164]: :+1: Sounds good. I think that warrants a changeset, and then this is ready to go!

File: packages/kit/src/core/config/index.spec.js [hunk 2]
```diff
@@ -240,7 +240,7 @@ test('fails if paths.base is not root-relative', () => {
 				}
 			}
 		});
-	}, /^config\.kit\.paths\.base option must be a root-relative path that starts but doesn't end with '\/'. See https:\/\/kit\.svelte\.dev\/docs\/configuration#paths$/);
+	}, /^config\.kit\.paths\.base option must either be the empty string, or a root-relative path that starts but doesn't end with '\/'. See https:\/\/kit\.svelte\.dev\/docs\/configuration#paths$/);
```
Comment (benmccann) [inline packages/kit/src/core/config/index.spec.js:243]:
```suggestion
	}, /^config\.kit\.paths\.base option must either be the empty string or a root-relative path that starts but doesn't end with '\/'. See https:\/\/kit\.svelte\.dev\/docs\/configuration#paths$/);
```

File: packages/kit/src/core/config/index.spec.js [hunk 3]
```diff
@@ -252,7 +252,7 @@ test("fails if paths.base ends with '/'", () => {
...
```
Comment (benmccann) [inline packages/kit/src/core/config/index.spec.js:255]:
```suggestion
	}, /^config\.kit\.paths\.base option must either be the empty string or a root-relative path that starts but doesn't end with '\/'. See https:\/\/kit\.svelte\.dev\/docs\/configuration#paths$/);
```

**Review by Conduitry — APPROVED** [review]

**Pull request merged**